src/Stats/utils.js

```diff
@@ -58,6 +58,14 @@ const filterData = (dataList, range, view) => {
   return result;
 };
 
+const filterType = (dataList, type) => {
```

Comment:
can we call it `filterTransactionByType` to be honest a stranger would never figure out what's going on in the application if we are full of `filterData`, `filterListView`, `filterType` and similar meaningful function names. But you won't know neither when you open the source code after 3 months.

Reply:
fixed